Automattic/newspack-plugin
# Fix build

### All Submissions:






### Changes proposed in this Pull Request:



It appears that some weirdness happened when git resolved the last couple merges (ActionCard component and newspack-components publishing). This moves some files around and changes some dependencies to make things work again. 

This should be merged before #69, and #69 should be brought up-to-date with the latest changes to prevent reintroducing these issues (there are merge conflicts there that need to be resolved anyways).

### How to test the changes in this Pull Request:

1. `npm run clean; npm run build:webpack` and `npm run test`
2. Verify Subscriptions Wizard and Components Demo work.
3. `cd assets/components; npm install; npm publish --dry-run` to verify the component packaging is still working well.

### Other information:
